hack/dashboard/assets/dashboard.json

```diff
@@ -52,77 +137,39 @@
           "color": {
             "mode": "thresholds"
           },
-          "custom": {
-            "align": "auto",
-            "displayMode": "auto",
-            "filterable": true,
-            "inspect": false
-          },
           "mappings": [],
           "thresholds": {
             "mode": "absolute",
             "steps": [
               {
                 "color": "green",
                 "value": null
-              },
-              {
-                "color": "red",
-                "value": 80
               }
             ]
           },
           "unit": "kwatth"
```

Comment:
isn't this kWh ?

Reply:
Yes, it's kWh only.

Reply:
but it reads `kwatth` is that expected?

Reply:
Yeah, Grafana adds unit as `kwatth` when we export the dashboard in JSON format.

Reply:
Thanks for the info, @vprashar2929 🤗